serializability-checker-compiler-plugin/src/test/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponentSpec.scala

```diff
@@ -88,6 +88,10 @@ class SerializabilityCheckerCompilerPluginComponentSpec extends AnyWordSpecLike
         testCode("AskHigherOrderClassicTest.scala", detectionType = 4)
       }
 
+      "RecipientRef type is used instead of ActorRef to reference an Actor" in {
+        testCode("AskRecipientRefTest.scala", detectionType = 1)
```

Comment:
Huh this `detectionType` is pretty unclear... pls replace with some enums or constants or whatnot while you're here

Reply:
done - please check out my solution